js/webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = [
         entry: './lib/extension.js',
         output: {
             filename: 'extension.js',
-            path: path.resolve(__dirname, '../bqplot/static'),
+            path: path.resolve(__dirname, '../share/jupyter/nbextensions/bqplot'),
```

Comment:
Why not just `../bqplot/nbextension`?

Changing it to `share/jupyter/nbextensions/bqplot` is a bit confusing IMO.

In my PR for JupyterLab 3 I create two directories under `bqplot`: `nbextension` and `labextension`.

Reply:
You should update `.gitignore` if we go for this approach. And we should do the same in all widgets libs.

Reply:
Because this is the location the file will end up (under) $prefix (see changes to setup.py), it makes it easier to digest where files go.

Reply:
(sorry, github was out of sync, you self answered already)

Reply:
The thing is I thought this was also used for the unpkg package, which is not installed in any user's path. But it turns out I was wrong, it's a separate target.